report-api/src/api/utils/logging.py

```diff
@@ -23,4 +27,40 @@ def setup_logging(conf):
         logging.config.fileConfig(conf)
         print(f'Configure logging, from conf: {conf}', file=sys.stdout)
     else:
+        try:
+            conf_path = resources.files('api') / 'logging.conf'
+            if conf_path.exists():
+                with NamedTemporaryFile(mode='w', suffix='.conf', delete=False) as tmp_file:
+                    tmp_file.write(conf_path.read_text())
+                    tmp_file.flush()
+                    logging.config.fileConfig(tmp_file.name)
+                    print(f'Configure logging from package resources: {conf_path}', file=sys.stdout)
+                    return
```

Comment:
probably not a good idea to have source control, control your logging? how will that work in DEV/TEST vs PROD?